app/src/main/java/com/yacgroup/yacguide/DescriptionActivity.kt

```diff
@@ -180,12 +173,20 @@ class DescriptionActivity : TableActivity() {
         _route?.typeOfClimbing?.takeIf { it.isNotEmpty() }?.let {
             layout.addView(WidgetUtils.createCommonRowLayout(this,
                     getString(R.string.type_of_climbing),
+                    "",
+                    WidgetUtils.infoFontSizeDp,
+                    View.OnClickListener { },
+                    Color.WHITE,
+                    Typeface.BOLD,
+                    40, 20, 20, 0))
```

Comment:
Is there a reason why you added more space on the left side for this view?

Reply:
Yes because this is just a detail. I find it a bit better arranged.

Reply:
Ok. Personally, I would left align the view but I'm fine with this.